### PR TITLE
ci: publish docs debug

### PR DESCRIPTION
**Pull request by gabreal**

### CODE REVIEW

File: scripts/gitlab/publish-docs.sh [hunk 1]
```diff
@@ -51,8 +51,11 @@ commit_files() {
 
 upload_files() {
     echo "__________Upload files__________"
-    git push -q origin HEAD
-    git push -q -f --tags
+    # this version of git (2.7.4) will dump the token on failure
```
Comment (kirushik) [inline scripts/gitlab/publish-docs.sh]: Why do we even use token instead of a deploy SSH key?

File: scripts/gitlab/publish-docs.sh [hunk 2]
```diff
@@ -51,8 +51,11 @@ commit_files() {
...
+    git push -q origin HEAD 2>&1 \
+      | sed -r "s|(${GITHUB_USER}):[a-f0-9]+@|\1:REDACTED@|g"
```
Comment (kirushik) [inline scripts/gitlab/publish-docs.sh]: I find it possible that git will change the failure format and this regexp will stop redacting the secret...
And this failure will be terribly hard to notice in advance...

Comment (gabreal) [reply scripts/gitlab/publish-docs.sh]: It's the most  simple pattern `username:token`

**Comment by TriplEight** [comment]
LGTM, also I'd vote moving to gitlab/github pages later

**Comment by gabreal** [comment]
thanks, I don't mind at all moving to something else when there is time to plan.

**Comment by soc1c** [comment]
labels!

**Pull request merged**